util/options_helper.cc

```diff
@@ -200,6 +201,31 @@ bool ParseBlockBasedTableIndexType(const std::string& type,
   return true;
 }
 
+bool SerializeEncodingType(
+  const EncodingType& type, std::string* value) {
+  switch (type) {
+    case EncodingType::kPlain:
+      *value = "kPlain";
```

Comment:
I am following the convention in this file. For example, "kBinarySearch" and "kHashSearch" are used instead of "BinarySearch" and "HashSearch". 
I would also prefer to use const string, but other common strings are used without a const string.
I would be prefer to make it into a separate fix with another checkin.